Reorder reingest dropdown options

sallain opened the pull request:
This was missed in the original PR, since I was using the processing
config for guidance. I've just switched the order of these two options
so that "Approve" is first, followed by "Reject", to be consistent with
all of the other dropdowns.

Connected to archivematica/Issues#819

sallain committed:
Reorder reingest dropdown options

This was missed in the original PR, since I was using the processing
config for guidance. I've just switched the order of these two options
so that "Approve" is first, followed by "Reject", to be consistent with
all of the other dropdowns.

Connected to archivematica/Issues#819

replaceafill reviewed (approved):
LGTM :+1: 

I also noticed `a6ed697e-6189-4b4e-9f80-29209abc7937` appears in https://github.com/artefactual/archivematica/blob/dddf0a6a74733463209bc5614a136b60c3f418a9/src/MCPServer/lib/assets/workflow.json#L8348-L8349 but I'm not sure how the user sees it?

replaceafill commented:
Ah! Maybe it's related to the watched directories functionality :man_shrugging: :

https://github.com/artefactual/archivematica/blob/dddf0a6a74733463209bc5614a136b60c3f418a9/src/MCPServer/lib/assets/workflow.json#L12268-L12273

sallain commented:
Good catch @replaceafill. That looks to me like a SIP approval step - it might be the one when you create a SIP from the Appraisal tab. I'll check!

sallain committed:
Change order of Approve SIP creation dropdown

This is the dropdown you get when you move something from backlog into
Ingest - microservice `Approve SIP creation`, `Job: Approve SIP
Creation`.

sallain commented:
@replaceafill It's the create-SIP-from-backlog approval dropdown, and it's now in yes/no order :smile:

replaceafill reviewed (approved):
Gracias!
